content/dart/concepts/type-conversion/terms/toStringAsFixed/toStringAsFixed.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: '.toStringAsFixed()'
+Description: 'Converts a numeric value into a string representation'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+  - 'Mobile Development'
+Tags:
+  - 'Dart'
+  - 'Data Types'
+  - 'Methods'
+  - 'Numbers'
+  - 'Strings'
+CatalogContent:
+  - 'learn-dart'
+  - 'paths/computer-science'
+---
+
+In Dart, **`.toStringAsFixed()`** is a method used to convert numeric [data types](https://www.codecademy.com/resources/docs/dart/data-types), typically used with `double`, to a string representation with a fixed number of decimal places. If the number is an integer it will be converted to a `double` before computing the string representation.
+
+## Syntax
+
+```pseudo
+doubleNumber.toStringAsFixed(fractionDigits);
+```
+
+- `doubleNumber`: Represents the number that we want to convert to `string`.
+- `fractionDigits`: is the number of digits to the right of the decimal point in the resulting string.
```

Comment:
```suggestion
- `numericValue`: Represents the numeric value to convert to a `string`. It can be either an `int` or a `double` data type.
- `fractionDigits`: An integer indicating the decimal places in the resulting string. It must be non-negative and will pad with zeros if greater than the actual decimal places in the number.
```